contracts/LockDealBundleProvider/LockDealBundleProviderState.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../LockDealNFT/LockDealNFT.sol";
+
+/// @title LockDealBundleProviderState contract
+/// @notice Contains storage variables
+contract LockDealBundleProviderState {
+    mapping(uint256 => bool) public isLockDealBundlePoolId;
+    mapping(uint256 => LockDealBundle) public poolIdToLockDealBundle;
+
+    struct LockDealBundle {
+        uint256 totalStartAmount;
```

Comment:
what if we don't save it, but calculate with a sum of params[0]
all provider got params[0]